src/igvfd/searches/configs/analysis_set.py

```diff
@@ -9,106 +9,194 @@ def analysis_set():
+                'description': 'The genome assembly associated with the analysis.',
+                'optional': True
+            },
+            'files.transcriptome_annotation': {
+                'title': 'Transcriptome Annotation',
+                'category': 'Analysis Set Details',
+                'description': 'The transcriptome annotation version of the analysis.',
+                'optional': True
             },
             'donors.taxa': {
                 'title': 'Taxa',
+                'category': 'Sample',
+                'description': 'The organism or species associated with the donor.'
             },
             'samples.classifications': {
-                'title': 'Classification'
+                'title': 'Classification',
+                'category': 'Sample',
+                'description': 'High-level classification of the sample.',
+                'optional': True
             },
             'samples.sample_terms.term_name': {
-                'title': 'Sample'
+                'title': 'Sample',
+                'category': 'Sample',
+                'description': 'The sample term name associated with the analysis set.'
             },
             'samples.targeted_sample_term.term_name': {
-                'title': 'Cellular Transformation Target'
+                'title': 'Cellular Transformation Target',
+                'category': 'Sample',
+                'description': 'Ontology term identifying the targeted endpoint sample resulting from differentation or reprogramming.'
             },
             'samples.disease_terms.term_name': {
                 'title': 'Disease',
+                'category': 'Sample',
+                'description': 'Ontology term of the disease associated with the sample.'
             },
             'samples.growth_medium': {
                 'title': 'Growth Medium',
+                'category': 'Sample',
+                'description': 'The growth medium or conditions used to culture the sample.',
+                'optional': True
             },
             'samples.modifications.modality': {
-                'title': 'Modification'
+                'title': 'Modification',
+                'category': 'Sample',
+                'description': 'The type of modification applied to the sample.',
+                'optional': True
             },
             'samples.treatments.treatment_term_name': {
-                'title': 'Treatment'
+                'title': 'Treatment',
+                'category': 'Sample',
+                'description': 'Treatments applied to the sample with the purpose of perturbation.',
+                'optional': True
             },
             'construct_library_sets.file_set_type': {
-                'title': 'Construct Library Data'
+                'title': 'Construct Library Data',
+                'category': 'Library',
+                'description': 'The type of the construct library associated with the analysis set.',
```

Comment:
'The type of the construct library associated with the analysis set, e.g, guide library, reporter library etc.'